content/influxdb/v2.0/upgrade/v1-to-v2/automatic-upgrade.md

```diff
@@ -101,10 +101,14 @@ You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scen
 ### User migration
 
 `influxd upgrade` migrates existing 1.x users and their permissions.
-However, it *does not migrate administrative users*.
+However, it **does not** migrate the following users:
 
-To review users with admin permissions, in the InfluxDB 1.x CLI, run `show users`.
-Any users labeled "admin" *will not* be migrated.
+- administrative (admin) users
+- non-admin users without privileges
```

Comment:
Users without admin permissions? 
Are admin users and non-admin users distinct in ways other than who has admin permissions? If yes, do we define this distinction?

Reply:
The 1.x docs make a clear divide between admin and non-admin: https://docs.influxdata.com/influxdb/v1.8/administration/authentication_and_authorization/#user-types-and-privileges

My brain likes something like this better:
```suggestion
- administrative (admin) users
- non-admin users that have not been granted any privileges
```
So if people want to make sure all non-admins are migrated, they see there's an action they can take (granting privileges) to make it happen